_projects/2025/100528259/100528259.Rmd

```diff
@@ -0,0 +1,986 @@
+---
+title: "An 8-Bit Look at the Evolution of BMI Values of Olympic Athletes"
+description: |
+  This study displays the evolution of BMI values of Olympic athletes who competed in the Summer Olympic Games of 1960 and 2016.
+categories: "2024"
+author: Gür Piren
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center")
```

Comment:
These are global options. So if you add `echo=TRUE` to this function call, you can remove it from every other chunk. ;-)